### PR TITLE
check if a user email exists

### PR DESCRIPTION
**Pull request by SimonLab**
see #464

@nelsonic can you have a look at this PR, thanks.

### CODE REVIEW

File: lib/database-helpers/elasticsearch/client_users/list_client_users.js [hunk 1]
```diff
@@ -5,11 +5,11 @@
 * @param {Function} callback - function with error and the list of client users
 */
 
-var clientES = require('../../../es.js');
+const clientES = require('../../../es.js');
 
 module.exports = (callback) => {
 
-  var numberClientUsers = 0;
+  let numberClientUsers = 0;
```
Comment (nelsonic) [inline lib/database-helpers/elasticsearch/client_users/list_client_users.js]: `let` and `var` are *identical* when used in function block.
see: http://stackoverflow.com/questions/762011/let-keyword-vs-var-keyword

**Comment by nelsonic** [comment]
@SimonLab code looks good. note comment on `let`. not an issue. just a note. 
merging. 👍

**Comment by SimonLab** [comment]
What is your view about let, var and const? I intend to not use anymore the var keyword. I don't see any cases where var can't be replaced by let or const. If we understand how to use let and const properly I don't think it should be a problem. What I understand:
- let when a variable can be reassigned, I use it often for counters
- const when a variable can't be reassigned
- The variable can be mutate in let and const, for example we can add a property to an object, it doesn't mean that it's a good thing (and I'd like to try to find a way to only use immutable variable)
- Both let and const are block scope

**Comment by nelsonic** [comment]
I have no _objection_ to using `let` or `const` in the correct context. 
However, to a *beginner* the `var` keyword is *much* clearer that it's a *variable* than `let`.
Try explaining `let` in one sentence... 

From the *examples* on MDN I can't think of a situation in *my* code where I would *want* to use a `let`... I don't use `for` loops and I don't see why I would want to define a variable inside an `if` block...
and I *certainly* would not advocate re-using the same variable *name* as in this example:

```js
function letTest() {
  let x = 31;
  if (true) {
    let x = 71;  // different variable
    console.log(x);  // 71
  }
  console.log(x);  // 31
}
```
If the person reading your code has to go and read a wikipedia article https://en.wikipedia.org/wiki/Let_expression or MDN post https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let before they understand why a certain keyword is used to prefix a variable instead of a more *obvious* one, then it's like using long/complicated ~~vocabulary~~ words instead of simple. 

[black-adder-c-confibularity](https://youtu.be/hOSYiT2iG08 "Click to watch it!")

> "_As a developer my code has to be clear for **everyone** reading it_"

The more **_complete_ beginner-friendly** you can make your code the better.
Using `let` *usually* raises more *questions* than it answers.
`const` is a *great* idea, implemented poorly. 

But... eric elliott said we should all be using `let` instead of `var` so we *must* comply...
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75 😆

**Pull request merged**